### PR TITLE
Don't close menu_popup when just a modifier key is pressed.

### PR DESCRIPTION
**Pull request by shalabhc**
Also add a list of modifiers to bindings.moon.

### CODE REVIEW

File: lib/howl/ui/menu_popup.moon [hunk 1]
```diff
@@ -46,6 +46,9 @@ class MenuPopup extends Popup
     page_up: => @list\prev_page!
 
     on_unhandled: (event, source, translations, self) ->
+      -- if a bare modifier such as just 'ctrl', don't close popup
+      return if bindings.is_modifier translations
+      -- any other not character such as home or escape closes the popup
```
Comment (refi64) [inline lib/howl/ui/menu_popup.moon]: nit: would `non-character` be better here?

Comment (shalabhc) [reply lib/howl/ui/menu_popup.moon]: `escape` is a non-character but that should close the popup (also `home`, `end`). Also we already have `.character` which is kind of the inverse of non-character. We really want to identify the modifiers like ctrl, shift, alt only.

**Comment by shalabhc** [comment]
@Janfel does this fix your iso shift modifier issue?

**Comment by Janfel** [comment]
Yes, it works perfectly with this fix.

**Pull request merged**